src/mixer/basetrackplayer.cpp

```diff
@@ -36,7 +36,8 @@ BaseTrackPlayerImpl::BaseTrackPlayerImpl(QObject* pParent,
           m_pConfig(pConfig),
           m_pEngineMaster(pMixingEngine),
           m_pLoadedTrack(),
-          m_replaygainPending(false) {
+          m_replaygainPending(false),
+          m_cloneFromChannel(NULL) {
```

Comment:
`NULL` is a legacy hack from before C++11. Please replace all new uses of `NULL` with `nullptr`